src/initialise.js

```diff
@@ -109,8 +109,13 @@ module.exports = async app => {
 		);
 
 		// Save each repository to our global map of repositories.
-		repositories.forEach(({ id, name }) => {
-			repositoryStore.set(id, { id, name });
+		repositories.forEach(({ id, name, archived }) => {
+			if (archived) {
+				logger.info(`Excluding ${name} (because it is )archived.`);
```

Comment:
Random `(` and `)` in this log message.

Reply:
![image](https://user-images.githubusercontent.com/51677/50823679-6b3a9180-132c-11e9-93e7-0f21625331b7.png)

😂

Reply:
![image](https://user-images.githubusercontent.com/51677/50823732-90c79b00-132c-11e9-9def-f86c6b669872.png)

😂

Reply:
(How )bizarre